mmdet/models/dense_heads/reppoints_head.py

```diff
@@ -534,6 +534,7 @@ def loss_single(self, cls_score, pts_pred_init, pts_pred_refine, labels,
         label_weights = label_weights.reshape(-1)
         cls_score = cls_score.permute(0, 2, 3,
                                       1).reshape(-1, self.cls_out_channels)
+        cls_score = cls_score.contiguous()
```

Comment:
`reshape` should have done `contiguous()` implicitly. 
Will there be bugs if this line is removed?

Reply:
> `reshape` should have done `contiguous()` implicitly.
> Will there be bugs if this line is removed?

yes , focal_loss will throw RuntimeError
```python
mmcv/ops/focal_loss.py", line 55, in forward
    input, target, weight, output, gamma=ctx.gamma, alpha=ctx.alpha)
RuntimeError: input must be contiguous
```